tests/common/test_persist_firewall_rules.py

```diff
@@ -215,6 +221,7 @@ def test_it_should_skip_setup_if_agent_network_setup_service_already_enabled_and
             self.__assert_systemctl_reloaded(validate_command_called=False)
             self.__assert_firewall_cmd_running_called(validate_command_called=True)
             self.__assert_firewall_called(cmd=FirewallCmdDirectCommands.QueryPassThrough, validate_command_called=False)
+            self.__assert_firewall_called(cmd=FirewallCmdDirectCommands.RemovePassThrough, validate_command_called=False)
```

Comment:
Can you add a case where you handle the scenarios on how the agent should react if the `remove_firewall` fails, for both iptables and firewalld. Thanks.

Reply:
added